brewcenter_api/brew_data/models/yeast.py

```diff
@@ -1,16 +1,32 @@
+from django.contrib.auth.models import User
 from django.db import models
 
+from brewcenter_api.accounts.models import Token
+
+
 class YeastType(models.Model):
     """
     Defines a Yeast type. Examples include:
     - Ale
     - Lager
     """
     name = models.CharField(max_length=255)
+
     def __str__(self):
         return self.name
 
 
+class YeastTypeRecommendation(models.Model):
+    model = models.ForeignKey(YeastType, null=True, blank=True)
+    accepted = models.BooleanField(default=False)
+    replaces = models.ForeignKey(YeastType, null=True, blank=True)
+    user = models.ForeignKey(User)
+    user_token = models.ForeignKey(Token)
```

Comment:
this attribute should be nullable because a user can either authenticate through jwt authentication, which we don't really care what their token was since it changes all the time. The only time we want to know what token was used is when the TokenAuthentication is used, which is when users can generate a token that lasts for life.